splitio/sync/manager.py

```diff
@@ -127,3 +128,44 @@ def _streaming_feedback_handler(self):
                 self._synchronizer.start_periodic_fetching()
                 _LOGGER.info('non-recoverable error in streaming. switching to polling.')
                 return
+
+class RedisManager(object):  # pylint:disable=too-many-instance-attributes
+    """Manager Class."""
+
+    def __init__(self, synchronizer):  # pylint:disable=too-many-arguments
+        """
+        Construct Manager.
+
+        :param unique_keys_task: unique keys task instance
+        :type unique_keys_task: splitio.tasks.unique_keys_sync.UniqueKeysSyncTask
+
+        :param clear_filter_task: clear filter task instance
+        :type clear_filter_task: splitio.tasks.clear_filter_task.ClearFilterSynchronizer
+
+        """
+        self._ready_flag = True
+        self._synchronizer = synchronizer
+
+    def recreate(self):
```

Comment:
?

Reply:
._ready_flag is called from SplitFactory class; resume method (see https://github.com/splitio/python-client/blob/telemetry-inmemory/splitio/client/factory.py#L267)
I was not sure if customers can implement the fork workaround with redis mode.

Reply:
sorry, i meant to ask what is this `recreate` method used for?

Reply:
Same as above, the method recreate is called  with the fork workaround implementation.